vision/detect_pdf.rb

```diff
@@ -0,0 +1,96 @@
+# Copyright 2018 Google, Inc
```

Comment:
@alixhami this is awesome, but not all vision samples have been migrated over to autogen library. This sample will most likely cause confusion with [existing samples](https://cloud.google.com/vision/docs/detecting-labels). 

@beccca is migration to Vision GAPIC prioritized by the ML APIs team for Q3?